Compile-and-Flash-STMBoard-Guide/README.adoc

```diff
@@ -0,0 +1,52 @@
+== *Compiling and Flashing STM Board*
+
+----
+This has been tested on Linux Mint and is expected to work on other Debian based distros
+----
+
+All commands will look like `this`
```

Comment:
This should be a given due to how asciidoc formats thing, this line is not necessary.